install_config/aggregate_logging.adoc

```diff
@@ -431,9 +431,14 @@ must be at least 512M. Possible suffixes are G,g,M,m.
 |A key/value map added to a PVC in order to select specific PVs.
 
 |`openshift_logging_es_pvc_dynamic`
-|If available for your
-cluster, set to `true` to have PVC claims annotated so that their
-backing storage is dynamically provisioned.
+|If available for your cluster, set to `true` to have PVC claims annotated so
+that their backing storage is dynamically provisioned. When set to `true` the
+storage class is ignored and not set to the PVC.
+
```

Comment:
If *what* is available for your cluster? 

The "When set to `true`... line doesn't make sense to me (doesn't mean it is wrong). There is a BZ comment that uses *added* to the PVC. That seems to make more sense to me.